jobs/build/check-bugs/Jenkinsfile

```diff
@@ -1,8 +1,21 @@
 #!/usr/bin/env groovy
 node {
+    currentBuild.displayName = params.BUILD_VERSION
+
+    // Skip check for 3.11
+    if (params.BUILD_VERSION == "3.11") {
```

Comment:
Think the skip for 3.11 should only happen for the regression check. We do want to have the check for bugs with the `blocker+` flag set.

Reply:
fixed